docs/api.rst

```diff
@@ -0,0 +1,267 @@
+
+The list of records is available at:
+
+* `$SERVER/buckets/build-hub/collections/releases/records <https://kinto-ota.dev.mozaws.net/v1/buckets/build-hub/collections/releases/records?_limit=10>`_
+
+And a single record at:
+
+* `$SERVER/buckets/build-hub/collections/releases/records/${ID} <https://kinto-ota.dev.mozaws.net/v1/buckets/build-hub/collections/releases/records/firefox_beta_50-0b11_macosx_el>`_
+
+A set of filters and pagination options can be used to query the list. The most notable features are:
+
+* querystring filters (with ``?field=value`` or dedicated operators like ``?min_field=value`` or ``?has_field=true``)
+* paginated list of records (follow the URL in the ``Next-Page`` response header)
+* fields selection (with ``?_fields=``)
+* polling for changes (with ``?_since=timestamp`` filter or ETags in request headers)
+
+`More information in the Kinto documentation <https://kinto.readthedocs.io/en/stable/api/1.x/filtering.html>`_.
+
+Elasticsearch API
+-----------------
+
+An ElasticSearch endpoint is also available for more powerful queries. It powers the online catalog.
+
+* `$SERVER/buckets/build-hub/collections/releases/search <https://kinto-ota.dev.mozaws.net/v1/buckets/build-hub/collections/releases/search>`_
+
+`More information in the Elasticsearch documentation <https://www.elastic.co/guide/en/elasticsearch/reference/current/search.html>`_
+
+
+Queries examples
+================
+
+Is this an official build id?
+-----------------------------
+
+In order to check that a build id exists, we'll just check that it is mentionned in at least one record.
+
+::
+
+    curl -s -I "${SERVER}/buckets/build-hub/collections/releases/records?build.id=20110110192031&_limit=1" | grep "Total-Records: 1"
+
+.. code-block:: javascript
+
+    import KintoClient from "kinto-http";
+    const client = new KintoClient(SERVER);
+    const collection = client.bucket("build-hub").collection("releases");
+    records = await collection.listRecords({limit: 1, filters: {"build.id": "20110110192031"}});
+    console.log(records.length == 1);
+
+.. code-block:: python
+
+    import kinto_http
+
+    client = kinto_http.Client("https://kinto-ota.dev.mozaws.net/v1")
+    records = client.get_records(**{"build.id": "20110110192031", "_limit": 1, "pages": 1},
+                                 bucket="build-hub", collection="releases")
+    print(len(records) == 1)
+
+What is the revision of a build id?
+-----------------------------------
+
+.. code-block:: python
+
+    client = kinto_http.Client("https://kinto-ota.dev.mozaws.net/v1")
+    records = client.get_records(**{"build.id": "20110110192031", "_limit": 1, "pages": 1},
+                                 bucket="build-hub", collection="releases")
+    try:
+        revision = records[0]["source"]["revision"]
+    except IndexError:
+        raise ValueError("Unknown build id")
+    except KeyError:
+        raise ValueError("Unknown revision")
+
+What are the locales of a specific version?
```

Comment:
Maybe something like `What locales are available for a certain version?` or `In what locales is a specific version available?`